ports/atmel-samd/common-hal/analogio/AnalogOut.c

```diff
@@ -65,12 +65,14 @@ void common_hal_analogio_analogout_construct(analogio_analogout_obj_t* self,
 
     #ifdef SAMD51
     hri_mclk_set_APBDMASK_DAC_bit(MCLK);
+    // This clock should be <= 12 MHz, per datasheet section 47.6.3.
     hri_gclk_write_PCHCTRL_reg(GCLK, DAC_GCLK_ID, GCLK_PCHCTRL_GEN_GCLK5_Val | (1 << GCLK_PCHCTRL_CHEN_Pos));
```

Comment:
Want to switch these GCLK values to CONF_GCLK_DAC_SRC to make sure they stay the same?

Reply:
Good idea!